.github/workflows/android-ci-pr-test.yml

```
@@ -2,9 +2,13 @@ name: Android CI Pull-Request Gradle Test
 
 on:
   pull_request:
+    # 'main' 또는 'android' 브랜치로 PR이 병합될 때 트리거
     branches:
       - main
       - android
+      # 'android' 디렉토리 내의 파일이 변경되었을 때만 워크플로우를 실행
+    path:
```

actionlint (1.7.7)

unexpected key "path" for "pull_request" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)